clickhouse-admin/types/src/lib.rs

```diff
@@ -900,6 +900,24 @@ impl KeeperConf {
     }
 }
 
+/// The configuration of the clickhouse keeper raft cluster returned from a
+/// single keeper node
+///
+/// Each keeper is asked for its known raft configuration via `clickhouse-admin`
+/// dropshot servers running in `ClickhouseKeeper` zones. state. We include the
```

Comment:
I know you just moved this struct, but there's a typo I made that you may as well cleanup while you're here.

```suggestion
/// dropshot servers running in `ClickhouseKeeper` zones. We include the
```